Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -2864,7 +2864,11 @@ public void CalculateActionFinalStatus(Act act)
                 act.Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
             }
 
-            if (act.Status != Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed && act.Status != Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed)
+            if(act.Status == Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped)
```

Comment:
If status is skipped already, whey we need to assign skipped again?

Reply:
done Changes